lib/src/us/kbase/kbasefeaturevalues/KBaseFeatureValuesImpl.java

```diff
@@ -117,13 +119,16 @@ public void estimateK(EstimateKParams params,
                 params.getNeighbSize(), params.getMaxItems());
         provenance.get(0).withDescription("K estimation for K-Means clustering method")
                 .withInputWsObjects(Arrays.asList(params.getInputMatrix()));
-        getWsClient().saveObjects(new SaveObjectsParams().withWorkspace(params.getOutWorkspace())
+	WorkspaceClient wsc = getWsClient();
+        List<Tuple11<Long,String,String,String,Long,String,Long,String,String,Long,Map<String,String>>> getInfo = wsc.saveObjects(new SaveObjectsParams().withWorkspace(params.getOutWorkspace())
```

Comment:
getInfo seems to be unused

Reply:
Looks like this happens in a number of places

Reply:
done see
https://github.com/kbaseapps/FeatureValues/pull/21/commits/e6322116fba34b1e3a0060a057ed35f65e9e0e15